eth_tester/backends/base.py

```diff
@@ -31,7 +33,7 @@ def time_travel(self, to_timestamp):
     # Mining
     #
     @abstractmethod
-    def mine_blocks(self, num_blocks=1, coinbase=None):
+    def mine_blocks(self, num_blocks=1, coinbase=ZERO_ADDRESS):
```

Comment:
Why did we need to change this from `None` -> `ZERO_ADDRESS`?

Reply:
I actually added a `coinbase` that should pass an address validation now. We can definitely change this to optional if we want but this just changes the default address to the zero address and passes the address validation. I could go either way here.

Reply:
On second thought, let me double check this... the validation was already there. I think this broke something early on.

Reply:
Ok yeah... either [this logic](https://github.com/ethereum/eth-tester/pull/245/files#diff-184f0343bc8af5da7a16faac90ae2ecde91970b2824e75699d9dd86fb69faf90L353-L358) needs to be repeated everywhere (e.g. `mine_block`... not a good solution), or we add the `None` check to the actual inbound validation method for this field, or we set it to the zero address and leave the validation alone. Can`coinbase` be empty on mainnet?

Reply:
@kclowes looks like we set `coinbase` to the zero address in py-evm as well, so atm coinbase is almost always the zero address unless it's passed in. I'm ok with either direction here. Let me know if you prefer one direction over another.

Reply:
Okay, I think it makes sense to leave it as the `ZERO_ADDRESS` then. Thanks for looking!